src/diffbot.js

```diff
@@ -96,7 +36,7 @@ class Diffbot {
       diffbot_url += `&fields=${options.fields.join(',')}`;
     }
 
-    if (options.discussion != undefined) {
+    if (options.discussion) {
```

Comment:
This fixes an issue where setting `options.discussion = false` has no effect
```suggestion
    if (options.discussion != undefined) {
```